Makefile

```diff
@@ -383,3 +406,4 @@ clean: $(GOLANGCI_LINT)
 	$(GOLANGCI_LINT) cache clean
 	chmod u+w -R $(LOCALBIN) # envtest makes its dir RO
 	rm -rf $(CLEANFILES)
+	rm $(PROMETHEUS_ALERT_TESTS:.unit-tests.yaml=.rules.yaml)
```

Comment:
Add it to CLEANFILES at the place it is produced. It will also fix the issue of  failure of removing of unexisting files if `make clean` run without the tests.